src/Atmos.jl

```diff
@@ -52,21 +52,93 @@ end # module SubgridScale
 
 module Microphysics
 
-export MP_n_0,
+export n0_rai,
+       n0_ice,
+       μ_sno,
+       ν_sno,
+       α_rai,
+       β_rai,
+       γ_rai,
+       δ_rai,
+       η_rai,
+       α_ice,
+       β_ice,
+       α_sno,
+       β_sno,
+       γ_sno,
+       δ_sno,
+       ζ_sno,
+       η_sno,
        C_drag,
        τ_cond_evap,
+       τ_sub_resub,
        q_liq_threshold,
        τ_acnv,
-       E_col,
-       a_vent,
-       b_vent,
+       E_lr,
+       E_ls,
+       E_ir,
+       E_is,
+       E_rs,
+       r_ice_snow,
+       a_vent_rai,
+       b_vent_rai,
+       a_vent_sno,
+       b_vent_sno,
        K_therm,
        D_vapor,
        ν_air,
        N_Sc
 
-"""Marshall-Palmer distribution n_0 coeff [1/m4]"""
-function MP_n_0 end
+"""Marshall-Palmer distribution n_0 coeff for rain [1/m4]"""
+function n0_rai end
+
+"""Marshall-Palmer distribution n_0 coeff for ice [1/m4]"""
+function n0_ice end
+
+"""coefficient to compute Marshall-Palmer distribution coefficient n_0 for snow [?]"""
+function μ_sno end
+
+"""coefficient to compute Marshall-Palmer distribution coefficient n_0 for snow [?]"""
```

Comment:
`[?]` 😱

Reply:
yes... There are 2 parameters that are empirical and have ridiculous  units. I'll discuss this with @tapios in the next microphysics meeting.  The alternative is to have a constant n0 for snow, but thats not what was done in the papers I read so far.

Reply:
fixed by dividing by rho_0=1kg/m3. Now all params have nice units.